client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -74,10 +74,24 @@ class VerificationCodeForm extends Component {
 		const isError = !! twoFactorAuthRequestError;
 
 		let helpText = translate( 'Type in the code generated by your Authenticator mobile application.' );
+		let labelText = translate( 'Verification Code' );
+		let smallPrint;
 		if ( twoFactorAuthType === 'sms' ) {
 			helpText = translate( 'A text message with the verification code was just sent to your phone number.' );
 		}
 
+		if ( twoFactorAuthType === 'backup' ) {
+			helpText = translate( "If you can't access your phone enter one of the 10 backup codes that were provided " +
+				'when you set up two-step authentication to continue.' );
+			labelText = translate( 'Backup code' );
+			smallPrint = (
+				<div className="two-factor-authentication__small-print">
+					{ translate( 'If you lose your device, accidentally remove the authenticator app, or are otherwise ' +
+						'locked out of your account, the only way to get back in to your account is by using a Backup Code.' ) }
```

Comment:
I think we probably don't want to capitalize `Backup Code` (we don't capitalize `verification code` in the other message).